demos/backend-workbench/_dockerfiles/director/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM continuumio/miniconda3
+LABEL maintainer="Sylvain Anderegg"
+
+RUN conda update conda
+RUN conda install flask
+RUN pip install docker
+
+WORKDIR /home/app
+
+ENV FLASK_APP=director.py
+ENV REGISTRY_AUTH = ''
+ENV REGISTRY_USER = ''
+ENV REGISTRY_PW = ''
+ENV REGISTRY_URL = ''
+ENV LOCAL_MODE='True'
+EXPOSE 8001
+COPY /director /home/app
+
+ENTRYPOINT python director.py
```

Comment:
**Prefer exec form**, i.e see https://docs.docker.com/engine/reference/builder/#entrypoint

ENTRYPOINT ["executable", "param1", "param2"]  

This comment applies to all ENTRYPOINT and CMD commands in Dockefiles

Reply:
done, but one needs to know that for stuff that needs environment variable resolutions you need to explicitely call ENTRYPOINT [ "sh" "-c", "the whole call string"] or it will not work.